modules/activiti-engine/src/test/java/org/activiti/engine/test/history/HistoricTaskInstanceUpdateTest.java

```diff
@@ -67,4 +72,41 @@ public void testHistoricTaskInstanceUpdate() {
     assertNull(historicTaskInstance.getDescription());
     assertNull(historicTaskInstance.getAssignee());
   }
+
+    @Deployment
```

Comment:
Tabs and spaces are mixed.

Reply:
Thanks for fix. I checked no Codacy issue.

I think there is no formatter when it is 5.x branch.

Activiti 7 has a formatter.
https://github.com/Activiti/activiti-build/tree/develop/ide-configuration/eclipse-configuration

Reply:
there is a formatter for eclipse here https://github.com/Activiti/Activiti/tree/5.x/eclipse